src/connection.js

```diff
@@ -865,7 +921,18 @@ class Connection extends EventEmitter {
     this.debug.payload(function() {
       return preloginPayload.toString('  ');
     });
-
+    if (this.fedAuthInfo.method != undefined) {
+      if (0 != preloginPayload.fedAuthRequired && 1 != preloginPayload.fedAuthRequired) {
+        this.emit('connect', ConnectionError(`Server sent an unexpected response for federated authentication value during negotiation. Value was  ${preloginPayload.fedAuthRequired}`, 'EFEDAUTH'));
+        return this.close();
```

Comment:
Can you fix the double space in error message? `Value was  __ ${preloginPayload.fedAuthRequired}`  :stuck_out_tongue_winking_eye: